### ISSUE TITLE
IPv6 testing

### ISSUE THREAD

**Issue by dtaht** [LibreQoE/LibreQoS#123]
It would be good to have an emulated environment simulating, say 20,000 customers, using a mix of ipv6 and ipv4, stressing things out with flent and/or trafficgen. 

My only thought further on this front today is choosing an ipv6 prefix to use, and fd1e:eb3e:005:: is the closest I can come to spelling LibreQos in hex semi-phonetically that I can think of.

**Comment by thebracket** [comment]
I really should add something to quietly prioritize `dead:beef` networks. :-p

That's honestly one of the hardest parts of working on the C/eBPF side of things - testing it at scale. It's not all that hard to look amazing between 3 VMs talking to one another. I have an ISP, but they frown upon "hey, let's try this version!" when it doesn't work so well...

**Comment by dtaht** [comment]
Imagine proving (to yourself, at least), that this code could and should run on every router in the world. I spent the first 2 years losing a ton of sleep, consulting with every expert I could find, as to what could possibly go wrong. Notably "drop head queueing" is unstable in most designs prior. We worried about attacks against the hash (still do), also... but yes, having a nice, hefty, easy to replicate set of vms and containers to run tests against would be nice, and if we are to convince people less crazy than us to have this code push every packet through their ISP, it would be helpful if we knew a given release or feature actually was working correctly. 

I also worry a lot about the ECN deployment. I figure a goodly percentage of ecn marked traffic is not getting a correct response from the endpoints...
